### PR TITLE
Update the version description in auto random document

### PR DESCRIPTION
**Pull request by glkappe**
https://asktug.com/t/topic/35498/5?u=%E6%88%B7%E5%8F%A3%E8%88%9F%E4%BA%A2

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### CODE REVIEW

File: auto-random.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ aliases: ['/docs-cn/dev/reference/sql/attributes/auto-random/']
 
 > **警告：**
 >
-> 当前 `AUTO_RANDOM` 属性为实验功能，**不建议在生产环境中使用**。在后续版本中，`AUTO_RANDOM` 的语法或语义可能会变化。
+> 当前 `AUTO_RANDOM` 属性于 v4.0.0 版本已为正式功能，**可以在生产环境中使用**。
```
Comment (breeswish) [inline auto-random.md:12]: Should we simply remove these lines for v4.0 and master branch?

Comment (glkappe) [reply auto-random.md:12]: > Should we simply remove these lines for v4.0 and master branch?

I think it's okay. At that time, I wanted to distinguish it from the experimental characteristics and give it a clear explanation.In future versions (such as 5.0) we can delete this line directly.

**Comment by tangenta** [comment]
The `auto_random` feature is not GA until release-4.0.3. We had better update the docs at that time.

**Comment by TomShawn** [comment]
@glkappe Please address the comments.

File: auto-random.md (hunk 1 above)
Comment (TomShawn) [inline auto-random.md:12]: @tangenta PTAL
```suggestion
> 当前 `AUTO_RANDOM` 属性于 v4.0.3 版本已为正式功能，**可以在生产环境中使用**。
```

Comment (tangenta) [reply auto-random.md:12]: LGTM

**Pull request merged**